text_handler.py

```diff
@@ -86,18 +88,45 @@ async def __handleMusicGrabber(self, message: types.Message):
                 await message.delete()
         except music_grabber.WrongFileSizeError as e:
             logger.error(str(e))
-            inform = await message.reply("This won't be downloaded because "
-                                         "its audio file size is greater "
-                                         "than 50M",
-                                         disable_notification=True)
-            await self.__deleteMessage(60, inform)
+            text = "This won't be downloaded because its audio file size " + \
+                "is greater than 50M"
+            inform = await message.reply(text, disable_notification=True)
+            await self.__delete_messages(60, (inform, ))
         except music_grabber.WrongCategoryError as e:
             logger.error(str(e))
         except Exception as e:
-            await message.reply("<b>Error</b>: <code>" + str(e) +
-                                "</code>", parse_mode=ParseMode.HTML,
-                                disable_notification=True)
+            markup = self.__make_inline_keyboard()
+            error_html = "<b>URL</b>: " + message.text + \
+                "\n<b>Error</b>: <code>" + str(e) + "</code>"
+            await message.reply(error_html, parse_mode=ParseMode.HTML,
+                                disable_web_page_preview=True,
+                                disable_notification=True, reply_markup=markup)
 
-    async def __deleteMessage(self, delay, message):
+    def __make_inline_keyboard(self):
+        markup = types.InlineKeyboardMarkup(reasize_keyboard=True)
+        item1 = types.InlineKeyboardButton("Report Error",
+                                           callback_data=ERROR_REPORT)
+        item2 = types.InlineKeyboardButton("Delete Only",
+                                           callback_data=ERROR_DELETE)
+        markup.add(item1, item2)
+        return markup
+
+    async def callback_error_message(self, callback: types.CallbackQuery):
+        from_chat_id = callback.message.chat.id
+        message_id = callback.message.message_id
+        callback_data = callback.data
+        if callback_data == ERROR_REPORT:
+            answer_text = "Successfully forward the error message to dev!"
+            await self.bot.forward_message(chat_id=DEV_ID,
+                                           from_chat_id=from_chat_id,
+                                           message_id=message_id)
+            await self.bot.answer_callback_query(callback.id, text=answer_text,
+                                                 show_alert=False)
+            await self.__delete_messages(3, (callback.message, ))
+        elif callback.data == ERROR_DELETE:
+            await callback.message.delete()
+
+    async def __delete_messages(self, delay, messages):
         await asyncio.sleep(delay)
-        await message.delete()
+        for msg in messages:
```

Comment:
why are you going through the list? Isn't there just ONE message?

Reply:
First change to 
```
    async def __delete_messages(self, delay, messages):
        await asyncio.sleep(delay)
        for msg in messages:
            await msg.delete()
```
to
```
    async def __delete_message(self, delay, message: types.Message):
        await asyncio.sleep(delay)
        await message.delete()
```

Then
in **callback_error_message** change 
```
await self.__delete_messages(3, (callback.message, ))
```
to
```
await self.__delete_message(DELETE_DELAY_IN_SEC, (callback.message[0]))
```

And finally in **__handle_music_grabber**

change
```
await self.__delete_messages(60, (inform, ))
```
to
```
await self.__delete_message(DELETE_SIZE_WARNING_DELAY, inform)
```

Reply:
there are three use of the method in this class, two of the three only delete one message, but for the "ping" and "pong" messages it's two. or should we change its behavior to pass one message only? I don't like to write that two long names twice, also I think there are may potentially have the use of delete multiple messages.

Reply:
*tuple
lists are mutable, tuples are immutable.

Since "ping" and "pong" are two messages and the method just delay and then delete message, if we use it twice, then we will feel the day between deleting this two messages. I can't think of other solution than using tuple/list and loop to delete one by one.

Reply:
Changed to this, looks good?
```
DELAY_DELETE_IN_SEC_PING = 5
DELAY_DELETE_IN_SEC_SIZE_WARNING = 60
DELAY_DELETE_IN_SEC_REPORT_ERROR = 3
```

it has the same prefix so that it's easy to recognize that those are used by the same method.

Reply:
yeah looks nice